app/src/Commands/Instances/ApplyCommand.php

```diff
@@ -113,16 +113,34 @@ public function execute(InputInterface $input, OutputInterface $output) : int
                return true;
             }));
 
+            $selection = [];
+            foreach ($states as $state) {
```

Comment:
The iteration variable **$state** has the same name as the variable **$state** on the function level, which might have unwanted side consequences. I'd suggest renaming the iterator to avoid confusion.

Reply:
done!